cmake/modules/FindPTex.cmake

```diff
@@ -27,6 +27,8 @@ if (WIN32)
             "$ENV{PROGRAMFILES}/Ptex/include"
             /usr/include
             DOC "The directory where Ptexture.h resides")
+    # TODO: figure out a way to add windows support for PTEX_PREFER_STATIC
+    # (since both static and dynamic libs can have a ".lib" suffix)
```

Comment:
You can do it two ways, quarantine by directory, or by filename. filename has the advantage that the versions can co-exist in one directory and there will never ever be ambiguity.

It's a serious issue because in general with gcc/clang, if you mix and match release and debug dylibs, in general nothing terrible will happen, modulo missing inlined symbols. On Windows, debug libs and release libs have separate heaps so you can end up in the amazing world of allocating in one heap and releasing to another, yielding absolute havoc.